default_system_message.txt

```diff
@@ -221,6 +221,22 @@ The prompt for the specific refactoring step may request "extra output fields" i
   ```
 ']. ['These should be formatted like the standard fields, above, using second-level Markdown header syntax delimited by single blank lines followed by the literal text (possibly multi-line) for their field values. These field names will be lower-case and may contain underscores.'])
 
+m5_if($md, ['
+  These should be formatted like the standard fields, above, using a second-level Markdown header ## extra_fields and then third-level headers for each key under it. For example:
```

Comment:
I see. Yes, you've prompted it for a literal `## extra_fields`. Prompt instead for 2nd level fields without `## extra_fields`. Then you shouldn't need the change above for manual extraction.